### PR TITLE
ROX-23972: Change system. to network. for netpols

### PR DESCRIPTION
**Pull request by ebensh**
Comments in https://github.com/stackrox/stackrox/pull/11586/files justify why

Tested:
```
go test -v github.com/stackrox/rox/pkg/helm/charts/tests/centralservices
go test -v github.com/stackrox/rox/pkg/helm/charts/tests/securedclusterservices
```

### CODE REVIEW

File: pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner-v4.test.yaml [hunk 1]
```diff
@@ -144,7 +144,7 @@ tests:
     .clusterroles["stackrox-scanner-v4-psp"] | assertThat(. == null)
 
 - name: "scanner with deprecated PodSecurityPolicies enabled"
-  values:
+  set:
```
Comment (mclasmeier) [inline pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner-v4.test.yaml:147]: This seems wrong to me.

**Review by mclasmeier — CHANGES_REQUESTED** [review]

File: pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner-v4.test.yaml (hunk 1 above)
Comment (mclasmeier) [inline pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner-v4.test.yaml:147]:
```suggestion
  values:
```

Comment (ebensh) [reply pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner-v4.test.yaml:145]: Fixed

**Review by mclasmeier — APPROVED** [review]

**Pull request merged**